addons/project/views/project_views.xml

```diff
@@ -1132,14 +1152,27 @@
                         <page name="task_dependencies" string="Blocked By" attrs="{'invisible': [('allow_task_dependencies', '=', False)]}" groups="project.group_project_task_dependencies">
                             <field name="depend_on_ids" nolabel="1" context="{'default_project_id' : project_id}">
                                 <tree editable="bottom">
+                                    <field name="legend_normal" invisible="1"/>
+                                    <field name="legend_done" invisible="1"/>
+                                    <field name="legend_blocked" invisible="1"/>
```

Comment:
I would place them just before `kanban_state` as they have been added for that reason.